presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -277,6 +277,74 @@ const HISTOGRAM_PROPERTIES = {
     disableHiddenCheck: true,
 };
 
+class RuntimeStatsList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: false
+        };
+    }
+
+    getExpandedIcon() {
+        return this.state.expanded ? "glyphicon-chevron-up" : "glyphicon-chevron-down";
+    }
+
+    getExpandedStyle() {
+        return this.state.expanded ? {} : {display: "none"};
+    }
+
+    toggleExpanded() {
+        this.setState({
+            expanded: !this.state.expanded,
+        })
+    }
+
+    renderMetricValue(unit, value) {
+      if (unit === "NANO") {
+          return formatDuration(parseDuration(value+ "ns"));
+      }
+      if (unit === "BYTE") {
+          return formatDataSize(value);
+      }
+      return formatCount(value); // NONE
+    }
+
+    render() {
+        return (
+             <table className="table" id="runtime-stats-table">
+                 <tbody>
+                 <tr>
+                     <th className="info-text">Metric Name</th>
+                     <th className="info-text">Sum</th>
+                     <th className="info-text">Count</th>
+                     <th className="info-text">Min</th>
+                     <th className="info-text">Max</th>
+                     <th className="expand-charts-container">
+                         <a onClick={this.toggleExpanded.bind(this)} className="expand-stats-button">
+                             <span className={"glyphicon " + this.getExpandedIcon()} style={GLYPHICON_HIGHLIGHT} data-toggle="tooltip" data-placement="top" title="Show metrics" />
+                         </a>
+                     </th>
+                 </tr>
+                 {
+                   Object
+                     .values(this.props.stats)
+                     .sort((m1, m2) => (m1.name.localeCompare(m2.name)))
+                     .map((metric) =>
+                         <tr style={this.getExpandedStyle()}>
```

Comment:
Just FYI, in my earlier PR, I added [clsx](https://www.npmjs.com/package/clsx), and if you use it here, it would be:
 ```suggestion
                         <tr className={clsx(!this.state.expanded && 'hide')}>
```
Your current code is good for me too.

Reply:
that's the short form (Strings style) of using clsx, this other form (Objects style) is like this: `clsx({'hide': !this.state.expanded})` and has better readability. It's handy if you have multiple classes and different conditions for each class. As for a single class, I usually just use the Strings form.

Reply:
Got it. I read a little bit more about this in the morning as well. It starts to make sense to me. Will keep this in mind. Appreciate you taking the time to educate me.